### ISSUE TITLE
Please add tips on how to add your package to Debian/Ubuntu repos

### ISSUE THREAD

**Issue by igor-petruk** [mmstick/cargo-deb#110]
Hi,

Thanks for the tool, it is really easy to run "cargo deb" and get our deb files out. Some like me configured their CI to build deb files and publish them to GitHub Releases.

It would be great to extend a README/Wiki with some tutorial where to move next. *This will really multiply the usefulness of this tool.*

Documentation on Debian/Ubuntu is long and is optimized to be most generic and apply to all sorts of packages. It would be great to write a shorter guide on how to publish your Rust apps to popular distro repos, at least Debian/Ubuntu and maintain them there. I am sure with this tool the number of steps is much smaller and you don't need to handcraft so many files manually.

I don't know if authors of this tool have an experience maintaining Debian packages with cargo deb, if yes, please share your experience with us.

Thanks!

**Comment by kornelski** [comment]
This project can't be used to make a package for the official Debian/Ubuntu distribution, because it doesn't follow Debian's requirements to produce source packages first.

**Comment by igor-petruk** [comment]
Yes, I've found this team https://salsa.debian.org/rust-team/debcargo-conf. One idea is to add the link clarifying the distinction.
